### PR TITLE
#19 Adding method to get branches for repository

### PR DESCRIPTION
**Pull request by jrolstad**
Added the Get-GitHubRepositoryBranch function to get all branches for a given repository

### CODE REVIEW

File: GitHubAnalytics.psm1 [hunk 1]
```diff
@@ -971,6 +971,54 @@ function Get-GitHubOrganizationRepository
     return $resultToReturn
 }
 
+function Get-GitHubRepositoryBranch
```
Comment (KarolKaczmarek) [inline GitHubAnalytics.psm1]: Please add comment with description of this function (SYNOPSIS etc.) similar to what other functions have.

**Comment by KarolKaczmarek** [comment]
Thanks for contributing @jrolstad 
I've reviewed the code and it looks good. Before merging we would require you to sign the CLA: https://cla.microsoft.com. 

Also, can you please add tests for the function you added?

Thanks!

**Comment by jrolstad** [comment]
Thanks for the feedback @KarolKaczmarek .  I've added tests and documentation.  I also signed the CLA, let me know if there is anything else.

**Comment by KarolKaczmarek** [comment]
Thanks @jrolstad! Everything looks good, however looks like CLA didn't go through for some reason (cla-required label is still there).

Could you please try resending it? Sorry for the inconveniences...

**Comment by jrolstad** [comment]
Hi @KarolKaczmarek , I just re-signed the CLA, everything should be good now

**Comment by KarolKaczmarek** [comment]
Thanks again @jrolstad!

**Pull request merged**